docs/source/reference/integration.rst

```diff
@@ -6,7 +6,7 @@ optuna.integration
    :nosignatures:
 
    optuna.integration.ChainerPruningExtension
-   optuna.integration.ChainerMNStudy 
+   optuna.integration.ChainerMNStudy
```

Comment:
Sorry, unrelated change. I believe it's acceptable but let me know if I should separate it out.